packages/editor-ui/src/composables/useBeforeUnload.ts

```diff
@@ -17,42 +15,40 @@ import { useWorkflowsStore } from '@/stores/workflows.store';
 export function useBeforeUnload({ route }: { route: ReturnType<typeof useRoute> }) {
```

Comment:
It would actually make more sense to rename this into `useCanvasBeforeUnload` since this contains business logic and split out a generic `useBeforeUnload` into a separate composable. No need to do here, I can also do it as a follow up.